### PR TITLE
added check to replace non-element atom labels

### LINKED ISSUE #92: Add check for atom labels that are not elements in atomistic parsers
Replace atom labels with 'X' for non-element labels. nb - this task should be moved to the normalizer in the new schema!

### PR DESCRIPTION
**Pull request by JFRudzinski**
@ladinesa added some hacky checks to ensure that atom labels which do not correspond to chemical elements are replaced by `X` to prevent issues in the normalizers. We discussed this in our Area C meeting today, and plan to implement more rigorous support for non-atomic systems within the new schema. Also, this should be checked in the system normalizer. But for now we enforce it in the parsers. I put a TODO there to move this check to the normalizer in the new schema.

Please give my code a check :pray:

### CODE REVIEW

File: atomisticparsers/h5md/parser.py [hunk 1]
```diff
@@ -580,6 +580,15 @@ def parse_system(self):
             time = system_info[step].pop("time")
             atoms_dict = system_info[step]
 
+            atom_labels = atoms_dict.get("labels")
+            if atom_labels is not None:
+                try:
+                    from ase import Atoms as ase_atoms
```
Comment (ladinesa) [inline atomisticparsers/h5md/parser.py]: maybe move to general imports

Comment (JFRudzinski) [reply atomisticparsers/h5md/parser.py]: done.

File: atomisticparsers/h5md/parser.py [hunk 2]
```diff
@@ -580,6 +580,15 @@ def parse_system(self):
...
+
+                    _ = ase_atoms(symbols=atom_labels)
```
Comment (ladinesa) [inline atomisticparsers/h5md/parser.py]: the specific function to test if symbols are valid in ase is 
```python
from ase.symbols import symbols2numbers
try:
    symbols2numbers(labels)
except KeyError:
    labels = ['X'] * len(labels)

Comment (ladinesa) [reply atomisticparsers/h5md/parser.py]: but I am also fine with trying to generate the full ase atoms

**Review by ladinesa — APPROVED** [review]

File: atomisticparsers/h5md/parser.py (hunk 2 above)
Comment (JFRudzinski) [reply atomisticparsers/h5md/parser.py]: ah, nice, thanks. done.

**Pull request merged**